db/install.xml

```diff
@@ -50,8 +50,10 @@
       </FIELDS>
       <KEYS>
         <KEY NAME="primary" TYPE="primary" FIELDS="id"/>
+        <KEY NAME="fk_page" TYPE="foreign" FIELDS="pageid" REFTABLE="mootimeter_pages" REFFIELDS="id"/>
```

Comment:
Please do not edit the install.xml manually, but instead use the xmldb editor.

By doing this this would also bump the "VERSION" string in the second line of the file to the latest date.

Also take care of the naming conventions: `Key and index names under the XMLDB Files must be formed by concatenating the name of the fields present in the key/index with the '"-" (minus) character.`, for details and other conventions see https://moodledev.io/general/development/tools/xmldb#conventions